piton/verif/env/metro_tile/metro_tile.v.pyv

```diff
@@ -235,9 +344,126 @@ always #25000 spi_sys_clk = ~spi_sys_clk;                       // 20MHz
 // SIMULATED BOOT SEQUENCE
 ////////////////////////////////////////////////////////
 
+int           rank; 
+int           size; 
+int           dest;
+byte unsigned valid_aux;
+
+int           tile_x; 
+int           tile_y;
+int           rankN; 
+int           rankE; 
+int           rankS; 
+int           rankW;
+
+
+int YUMMY_NOC_1 ;
+int DATA_NOC_1  ;
+int YUMMY_NOC_2 ;
+int DATA_NOC_2  ;
+int YUMMY_NOC_3 ;
+int DATA_NOC_3  ;
+
 `ifndef VERILATOR
 initial
 begin
+    string filerank;
+
+    YUMMY_NOC_1 = 0;
+    DATA_NOC_1  = 1;
+    YUMMY_NOC_2 = 2;
+    DATA_NOC_2  = 3;
+    YUMMY_NOC_3 = 4;
+    DATA_NOC_3  = 5;
+
+    //metro initialization
+    initialize();
+    //barrier();
+    rank = getRank();
+    size = getSize();
+    filerank.itoa(rank);
+    $dumpfile({"metro_tile_",filerank,".vcd"});
```

Comment:
Does this work as intended?

Reply:
Yes, str.itoa() works as intended and I've verified it produces the correct vcd files.